src/Behat/Behat/Tester/Cli/RerunController.php

```diff
@@ -92,12 +95,12 @@ public function execute(InputInterface $input, OutputInterface $output)
 
         $this->key = $this->generateKey($input);
 
-        if (!$input->getOption('rerun')) {
+        if (!$input->getOption('rerun') && !$input->getOption('rerun-only')) {
             return;
         }
 
         if (!$this->getFileName() || !file_exists($this->getFileName())) {
-            return;
+            return $input->getOption('rerun-only') ? 0 : null;
```

Comment:
Please add an output message here to explain the early termination (and update the feature accordingly).